src/main/java/spireMapOverhaul/zones/brokenSpace/BrokenSpaceZone.java

```diff
@@ -0,0 +1,254 @@
+        super.renderOnMap(sb, alpha);
+        BrokenSpaceRenderPatch.StopFbo(sb);
+        if (alpha > 0) {
+            FontHelper.renderFontCentered(sb, FontHelper.menuBannerFont, name,
+                    labelX * SPACING_X + OFFSET_X, labelY * Settings.MAP_DST_Y + OFFSET_Y + DungeonMapScreen.offsetY,
+                    Color.WHITE.cpy(), 0.8f
+            );
+        }
+    }
+
+    @Override
+    public boolean generateMapArea(BetterMapGenerator.MapPlanner planner) {
+        return generateNormalArea(planner, width, height);
+    }
+
+    @Override
+    protected boolean canIncludeEarlyRows() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeFinalCampfireRow() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeTreasureRow() {
+        return false;
+    }
+
+    @Override
+    public boolean canSpawn() {
+        return AbstractDungeon.actNum >= 2;
+    }
+
+    @Override
+    public Color getColor() { //I considered changing this to a variable, but a method lets you do funky stuff like a rainbow zone that changes colors or something.
+        return color;
+    }
+
+    public AbstractCard getTrulyRandomCard(Random rng) {
+        AbstractCard.CardRarity rarity = AbstractCard.CardRarity.COMMON;
+
+        // pick RANDOM rarity
+        int roll = rng.random(7);
+        switch (roll) {
+            case 0:
+            case 1:
+                rarity = AbstractCard.CardRarity.COMMON;
+                break;
+            case 2:
+            case 3:
+                rarity = AbstractCard.CardRarity.UNCOMMON;
+                break;
+            case 4:
+            case 5:
+                rarity = AbstractCard.CardRarity.RARE;
+                break;
+            case 6:
+                return AbstractDungeon.returnRandomCurse().makeCopy();
+
+        }
+
+
+        AbstractCard c = CardLibrary.getAnyColorCard(rarity).makeCopy();
+
+        return c;
+
+
+    }
+
+    @Override
+    public ArrayList<AbstractCard> getAdditionalCardReward() {
+        ArrayList<AbstractCard> cards = new ArrayList<>();
+        int amount = 3;
+
+        for (AbstractRelic r : AbstractDungeon.player.relics) {
+            amount = r.changeNumberOfCardsInReward(amount);
+        }
+
+        for (int i = 0; i < amount; i++) {
+            AbstractCard c = getTrulyRandomCard(AbstractDungeon.cardRandomRng);
+            cards.add(c);
+            UnnaturalCardField.unnatural.set(c, true);
+
+        }
+        return cards;
+    }
+
+    @Override
+    public void modifyReward(RewardItem rewardItem) {
+        if (rewardItem.type == RewardItem.RewardType.RELIC) {
+            AbstractRelic origRelic = rewardItem.relic;
+            AbstractRelic newRelic = getValidBrokenRelic();
+
+            if (newRelic != null) {
+                rewardItem.relic = newRelic;
+                rewardItem.text = newRelic.name;
+
+                switch (origRelic.tier) {
```

Comment:
Hmm, this might crash when the pool is empty and circlets show up

Reply:
Useful information: Circlet's are `RelicTier.SPECIAL`, which is the rarity also used for event relics.

I suggest adding the following below:
```
case SPECIAL:
    // There's no pool for special relics, so we don't need to do anything
    break;
```

I think it's fine to do nothing for Circlets (and for any other special rarity relic that somehow shows up as an elite drop), but it's good to show that it's been specifically thought about, and I wouldn't want the log statement in the `default` case to show up.

Reply:
pushing fix.